### ISSUE TITLE
Can I directly replace the network card on the motherboard with BCM94360?

### ISSUE THREAD

**Issue by aahowe** [glekner/Z690I-AORUS-OC-EFI#9]

**Comment by glekner** [comment]
yes, but only do so if you are experienced with disassembling motherboards

**Comment by aahowe** [comment]
thanks. I found that the network card that comes with the motherboard is based on the cnvi protocol. I thought it would not be recognized if it was replaced with an ngff network card.

**Comment by aahowe** [comment]
After testing, replacing the onboard ax201 Nic with bcm94352z cannot drive Wi-Fi after entering the Windows system.

**Comment by mistercorea** [comment]
With the BCM94360NG card, which antenna are you using? I was using the one came with the mother board but I am getting bad signal from both bluetooth and wifi

**Comment by aahowe** [comment]
I have no problem using the antenna that comes with the motherboard. Check if the antenna is not connected properly.

**Comment by mistercorea** [comment]
~~Don't you have a problem consistently getting disconnected from devices? if not what is the best way to troubleshoot this?~~

I take it back upgrading the Bluetooth context to latest solved the problem

**Comment by mistercorea** [comment]
These are hardware info I have from sysinfo.

![Screen Shot 2023-03-08 at 11 57 35 AM](https://user-images.githubusercontent.com/2279899/223792456-e9b228a0-2664-4507-8a75-211d6abb1a53.png)
![Screen Shot 2023-03-08 at 11 57 20 AM](https://user-images.githubusercontent.com/2279899/223792459-13058ffb-0918-49cc-9dcd-348441c47e4f.png)
